esutil/bulk_indexer.go

```diff
@@ -631,6 +636,8 @@ func (w *worker) flushBuffer(ctx context.Context) error {
 		}
 	}
 
+	atomic.AddUint64(&w.bi.stats.flushedBytes, uint64(bufLen))
```

Comment:
Nit: I'd move it above the if, for me that's easier to read

Reply:
Do you mean above the for loop?

Reply:
I meant something like line 522, back where `bufLen` is set. But  I missed that there are a bunch of returns in the meantime so we might not get to this line. Let's keep as is